src/logic/__test__/calculator-operator.test.js

```diff
@@ -0,0 +1,56 @@
+// import { render, screen } from '@testing-library/react';
+import calculator from '../calculate';
+import operate from '../operate';
```

Comment:
### SUGGESTION
It's a good practice to write tests separately for every single file. Please improve on this.

Reply:
Alright I will work on that, thanks